packs/ruby/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:onbuild
+FROM ruby:latest
 
```

Comment:
Are we able to fix this tag to a version?  We've seen issues in the past when using `latest` as two builds can pull different layers and have different outcomes.  If not we could pull the ,current `:latest` and push up a tagged version to our jenkinsxio registry and use that instead?

Reply:
@rawlingsj we definitely can stick to a version. What does that mean though? that we will need to have multiple build packs for the ruby versions that we support? it's very common in Ruby land to have the last 3 minor versions and a bunch of minor versions available. Pretty much everything that's not EOL. Or will Jx only provide one version and it's the user's responsibility to provide their own if they differ from it?

Reply:
yeah I think the latter sounds like a good approach TBH, providing one version for now, we do that with the nodejs buildpack https://github.com/jenkins-x-buildpacks/jenkins-x-kubernetes/blob/master/packs/javascript/Dockerfile#L1